README.md

```diff
@@ -146,6 +149,41 @@ In the above example `results` is an object with the following structure:
     },
 }
 ```
+#### `resizeBrowserToBodyHeight`
+Is a basic command, it take one optional parameter `offset` and does what it says.
+
+As native screenshots only capture the visible viewport area, this increases the browser height to match the full body modified by any `offet` value provided. This should allow large elements to be fully captured, including the body element.
```

Comment:
Typo, `offet` -> `offset`

Reply:
corrected from `offset/offet` to `adjustment`